### PR TITLE
Clean up address usage (part 1 of many)

### PR DESCRIPTION
**Pull request by pattyshack**
Prep work for changing address internal representation

The number of changes is getting a bit out of hand.  I'll split up the change into smaller PRs, with the goal of consistently operating on flow.Address whenever possible.

Note that (except for tests) I'm trying to consistently name the address variables based on their types:
 - address is always a flow.Address
 - runtimeAddress is a common.Address
 - cadenceAddress is a cadence.Address

### CODE REVIEW

File: fvm/environment/transaction_info.go [hunk 1]
```diff
@@ -150,8 +150,8 @@ func (info *transactionInfo) LimitAccountStorage() bool {
 	return info.params.LimitAccountStorage
 }
 
-func (info *transactionInfo) SigningAccounts() []common.Address {
-	return info.authorizers
+func (info *transactionInfo) SigningAccounts() []flow.Address {
+	return info.params.TxBody.Authorizers
```
[the remark this answers is not in the document]
Comment (pattyshack) [reply fvm/environment/transaction_info.go:154]: GetSigningAccounts is an cadence runtime interface api, so we can't easily rename that.  I'll rename SigningAccounts to hopefully reduce confusion

[the remark this answers is not in the document]
Comment (pattyshack) [reply fvm/environment/transaction_info.go:154]: I've decided to remove `SigningAccounts()` instead.  It's only used by contract updater, and contract updater can directly access the tx body

**Review by zhangchiqing — APPROVED** [review]

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors retry

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**